digdag-server/src/main/java/io/digdag/server/rs/AttemptResource.java

```diff
@@ -256,10 +258,13 @@ public Response startAttempt(RestSessionAttemptRequest request)
                     .transform(r -> collectResumingTasks(r))
                     .or(ImmutableList.of());
 
+            Config params = request.getParams();
+            putAdditionalParams(params);
```

Comment:
IIUC `request.params` store user defined parameters.
So mix of system and user defined may be harmful.

Reply:
Ah scheduler may write some parameters to it. Let me check it.

Reply:
Ok, I confirmed both user define and system defined params are stored.

Reply:
so, then `params.set("__td__.user_id", info.get("id", Integer.class));` or some.